internal/webhook/external/common_test.go

```diff
@@ -16,3 +16,9 @@ func sign(content []byte) string {
 		hex.EncodeToString(mac.Sum(nil)),
 	)
 }
+
+func signWithArtifactory(content []byte) string {
+	mac := hmac.New(sha256.New, []byte(testSigningKey))
+	_, _ = mac.Write(content)
+	return hex.EncodeToString(mac.Sum(nil))
+}
```

Comment:
Since this signing function is, at least for now, unique to Artifactory, I'd consider moving it into `artifactory_test.go`.

Alternatively, and perhaps better, you could leave it here, but rename it to something like `signWithoutAlgoPrefix()` and then it seems we could potentially reuse it in other future cases and being here in `common_test.go` would be fine because it would no longer seem so Artifactory-specific.